pithos/pithos.py

```diff
@@ -22,13 +22,24 @@
 import signal
 
 import gi
-gi.require_version('Gst', '1.0')
 from gi.repository import Gst, GstPbutils, GObject, Gtk, Gdk, Pango, GdkPixbuf, Gio, GLib
 import contextlib
 import html
 import math
 import urllib.request, urllib.error, urllib.parse
 import json
+
+try:
+    gi.require_version('Gst', '1.0')
+except ValueError, e:
+    if e[0].startswith("Namespace Gst not available for version"):
+        print "Received error %s from Gstreamer." % e[0]
```

Comment:
Except it does use python3 by default and you had to call it with python2 manually... https://github.com/pithos/pithos/blob/master/setup.py#L1

Also setuptools is absolutely useless for checking for deps and since they are runtime deps it would be counter productive for packagers to force deps anyway.